test/WebJobs.Extensions.Tests/Extensions/Timers/Scheduling/CronScheduleTests.cs

```diff
@@ -54,7 +55,7 @@ public void GetNextOccurrence_ThreeDaySchedule_MultipleScheduleIterations()
         [Fact]
         public void Interval_IntoDST_ReturnsExpectedValue()
         {
-            TimerListenerTests.SetLocalTimeZoneToPacific();
+            using var timeZoneSetter = TimeZoneSetter.PacificStandard;
```

Comment:
It is possible to make these theories and test them all on a -, + and 0 timezone?

Reply:
We could, but it'd take some work to identify the dates for some other time zones. And it may not be worth it as whether it's +/- UTC doesn't matter to this logic as much b/c they're not at the "edges" of min/max values like DateTime.Min/Max are (and we never use DateTime.Max anywhere, which is why we have no test for it).